src/main/java/swaglabs/common/CommonMethods.java

```
@@ -63,7 +65,7 @@ public <T extends WebElement> void click(T element) {
 	 * @param text
 	 * @param timeoutSeconds
 	 */
-	public <T extends WebElement> void waitForElementContainText(T element, String text, Integer timeoutSeconds) {
+	public static <T extends WebElement> void waitForElementContainText(T element, String text, Integer timeoutSeconds) {
 		boolean check = false;
 		for (int i = 0; i < timeoutSeconds; i++) {
 			try {

```

Empty catch block is a bad practice. At minimum, log the exception or add a comment explaining why it's intentionally empty.